singularity-eos/eos/eos_builder.cpp

```diff
@@ -65,6 +110,14 @@ EOS EOSBuilder::buildEOS(EOSBuilder::EOSType type,
       int matid = mpark::get<int>(base_params["matid"]);
       if (type == EOSType::SpinerEOSDependsRhoT) {
         SpinerEOSDependsRhoT s(filename,matid,reproducibility_mode);
+	if (units) {
```

Comment:
Could you fix the indentation to match the control flow? These `if (units) {...}` blocks are in the scope of the `if (type == EOSType::SpinerEOSDependsRhoT  || type == EOSType::SpinerEOSDependsRhoSie) {` block but the indentation makes them look to be in a different scope.

Reply:
oof yes good catch. Not sure what's going on with the indentation here.

Reply:
Hmmmmm... I just took a closer look and the indentation is all over the place in this file. It may be a harder fix.

I'm just fixing it on my end for now and maybe I can make a separate MR since it touches just about every line in the file and I don't want to pollute the real changes here.

Reply:
That works too! My editor did a reasonable job at trying to fix the indentation but it still required some hand fixing. I'm all in favor of automation :-)

Reply:
Will do. I'll submit a separate PR from develop for that. Then this one will look cleaner too.

Reply:
@jhp-lanl I merged in the clang-format. Diff should look much cleaner now.